lib/backend/model/statusreports.go

```diff
@@ -77,16 +82,22 @@ func (options ActiveStatusReportListOptions) KeyFromDefaultPath(ekey string) Key
 		log.Debugf("Didn't match regex")
 		return nil
 	}
-	name := r[0][1]
+	regionString := r[0][1]
+	name := r[0][2]
+	if options.RegionString != "" && regionString != options.RegionString {
+		log.Debugf("Didn't match region %s != %s", options.RegionString, regionString)
+		return nil
+	}
 	if options.Hostname != "" && name != options.Hostname {
 		log.Debugf("Didn't match name %s != %s", options.Hostname, name)
 		return nil
 	}
-	return ActiveStatusReportKey{Hostname: name}
+	return ActiveStatusReportKey{Hostname: name, RegionString: regionString}
 }
 
 type LastStatusReportKey struct {
-	Hostname string `json:"-" validate:"required,hostname"`
+	Hostname     string `json:"-" validate:"required,hostname"`
+	RegionString string
```

Comment:
Missing tags?

Reply:
(As above)

Reply:
I don't follow, shouldn't this have a validate tag even if the json tag isn't relevant?